plugin/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+	"root": true,
+	"extends": "wpcalypso",
+	"parser": "babel-eslint",
+	"env": {
+		"browser": true,
+		"node": true
+	},
+	"parserOptions": {
+		"ecmaVersion": 3
```

Comment:
Likely ignored with `babel-eslint` parser override. Do we need to use `babel-eslint` or is the default parser sufficient?